src/routes/cache-hooks/guards/pre-execution.guard.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  CanActivate,
+  ExecutionContext,
+  Inject,
+  Injectable,
+} from '@nestjs/common';
+import { ILoggingService, LoggingService } from '@/logging/logging.interface';
+import { formatRouteLogMessage } from '@/logging/utils';
+
+/**
+ * The PreExecutionLogGuard guard outputs a log line containing the request data.
```

Comment:
nitpick: the only request data that is being logged is the path. It does not include the body of the response for example.

Reply:
I agree, thanks! [5c9cfb7](https://github.com/safe-global/safe-client-gateway/pull/1120/commits/5c9cfb7870538e0177c956a10ef91b3e7b9b620f)